src/client/pages/Events/EventsPage.jsx

```diff
@@ -1,9 +1,39 @@
-import React from 'react'
+import React, { Fragment } from 'react';
 
-class EventsPage extends React.Component {
-  render() {
-    return <h1>Events Page</h1>
-  }
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+
+import PageInfo from '../../components/PageInfo'
```

Comment:
This is where you would do
```suggestion
 import PageInfo from 'components/PageInfo'
```